app/components/page-wrapper/index.hbs

```diff
@@ -0,0 +1,8 @@
+<div
+  local-class='page-wrapper-root {{or @backgroundColor "dark"}}'
```

Comment:
can we rename this class to `page-wrapper-background-{{or @backgroundColor "dark"}}`

Reply:
Done